python/skupper_router/management/skrouter.json

```diff
@@ -623,6 +623,13 @@
                     "required": false,
                     "default": "auto",
                     "create": true
+                },
+                "dropTcpConnections": {
+                    "type": "boolean",
+                    "default": false,
```

Comment:
Since this PR is going to land on main which is going to be released as Skupper 2.0 (skupper-router 3.0), @grs suggested that the default for this flag can be true. 

Since 2.0 is a major version, we are not going to guarantee backward compatibility, so defaulting the flag to true is ok to do.

Reply:
When this PR is back ported to 1.8 version of Skupper, we will set the default on this flag to false, since that will ensure backward compatibility.

1.8 is the upcoming upstream release of Skupper

Reply:
I changed the default value to true.